classes/class-twentytwenty-customize.php

```diff
@@ -67,11 +67,71 @@ public static function register( $wp_customize ) {
 				)
 			);
 
+			/**
+			 * Implementation for the accent color.
+			 * This is different to all other color options because of the accessibility enhancements.
+			 * The control is a hue-only colorpicker, and there is a separate setting that holds values
+			 * for other colors calculated based on the selected hue and various background-colors on the page.
+			 *
+			 * @since 1.0.0
+			 */
+
+			// Add the setting for the hue colorpicker.
+			$wp_customize->add_setting(
+				'accent_hue',
+				array(
+					'default'           => 344,
+					'type'              => 'theme_mod',
+					'sanitize_callback' => 'absint',
+					'transport'         => 'postMessage',
+				)
+			);
+
+			// Add setting to hold colors derived from the accent hue.
+			$wp_customize->add_setting(
+				'accent_accessible_colors',
```

Comment:
This is a control-less setting. We're simply registering it here to make sure it gets properly sanitized with our callback. 
Since the setting doesn't have a control, we'll be manipulating its value from the JS API. It doesn't need a control, it simply needs to save the accessible colors that result from all the other controls.